docs/config-file/v1.rst

```diff
@@ -122,9 +122,9 @@ if they want to be on the cutting edge.
 Certain Python versions require a certain build image,
 as defined here:
 
-* ``1.0``: 2, 2.7, 3, 3.4
-* ``2.0``: 2, 2.7, 3, 3.5
-* ``latest``: 2, 2.7, 3, 3.3, 3.4, 3.5, 3.6
+* ``1.0``: :buildpyversions:`1.0`
+* ``2.0``: :buildpyversions:`2.0`
+* ``latest``: :buildpyversions:`latest`
```

Comment:
ConfigV1 and ConfigV2 share the same `build.image` possible values.

https://github.com/rtfd/readthedocs.org/blob/2bf6e9943c4177d11e125fd077a07f7168fce044/readthedocs/config/config.py#L252-L266

I suppose that we should update this here.